Description: Make target installation directory a command line option
 Use a variable for the install prefix, so we can specify it at build
 time on the command line.
Author: Julien Blache <jblache@apple.com>

Index: swiftlang-5.7.1/swift/utils/build-presets.ini
===================================================================
--- swiftlang-5.7.1.orig/swift/utils/build-presets.ini
+++ swiftlang-5.7.1/swift/utils/build-presets.ini
@@ -824,7 +824,7 @@ install-swiftpm
 install-swift-driver
 install-xctest
 install-libicu
-install-prefix=/usr
+install-prefix=%(install_prefix)s
 install-sourcekit-lsp
 install-swiftdocc
 build-swift-static-stdlib
